json_object.go

```diff
@@ -0,0 +1,45 @@
+package dl
+
+import (
+	"encoding/json"
+	"reflect"
+
+	"github.com/google/uuid"
+)
+
+// JSONObject is a general structure to represent JavaScript objects.
+type JSONObject map[string]interface{}
```

Comment:
Just wondering. Is it OK to use `JSON` from the style perspective?
(do we have google go style guide or something else?)

Reply:
@feeblefakie 
Yes.
It's ok.

It's mentioned and recommended in
https://github.com/golang/go/wiki/CodeReviewComments#initialisms
The acronyms part in namings should be either in all capitals or in all lower case.

○ userID
○ userid
× userId

○ HTTPServer (for exporting)
○ httpServer (for use inside the package)
× HttpServer

In our case, JSONObject will be exported to the other packages, so the JSONObject naming is fine.
(if it is used internally, then the jsonObject naming would be adopted)